src/Utils.ts

```diff
@@ -13,29 +12,44 @@ import {
     Timestamp,
     CHANNEL_TYPE
 } from './types'
-// import { WebStorageSessionStore } from 'matrix-js-sdk/lib/store/session/webstorage'
+import { IndexedDBStore, MemoryStore, createClient } from 'matrix-js-sdk'
+import { IStore } from 'matrix-js-sdk/lib/store'
+import { LocalStorage } from 'node-localstorage'
+
+import request from 'request'
```

Comment:
Is there a way to prevent these imports in the browser? Everything adds up for the final kernel bundle size

Reply:
yes, is causing some troubles, should be part of a dev dependency and use only for tests